src/graphql/data-sources/serlo.ts

```diff
@@ -206,6 +207,14 @@ export class SerloDataSource extends RESTDataSource {
     await this.environment.cache.set(cacheKey, response)
   }
 
+  public async getSubscription({
+    userid,
```

Comment:
```suggestion
    id,
```
for consistency with notifications.